a=3
b=83
print(a+b)

output:
/home/ansible-practice/PycharmProjects/test/venv/bin/python /home/ansible-practice/PycharmProjects/test/1.py
86

Process finished with exit code 0
======================================
a=3
b=83
print(a+b)
print(int.__add__(a,b))
output:
/home/ansible-practice/PycharmProjects/test/venv/bin/python /home/ansible-practice/PycharmProjects/test/1.py
86
86

Process finished with exit code 0
======================================
even we use just a+b internally it is calling __add__ method,for -, it calls __sub__ , for * , it calls __mul__ .
====================================================================================================================
